server.py

```diff
@@ -37,6 +37,11 @@ def download(path):
     return static_file(path, root=question_dir)
 
 
+@route('/static/<filepath:path>')
+def server_static(filepath):
+    return static_file(filepath, root='PyJudge/css')
```

Comment:
```suggestion
    return static_file(filepath, root=os.path.join(dir_path, 'static'))
```

Reply:
Didn't get why you used `root=os.path.join(dir_path, 'static')` Can you please explain?

Reply:
Of course I'll explain. But before that I'd like to know the reasoning behind putting `PyJudge/css` as the root.

Reply:
It is the path given when you find files in the repo and css is the folder so i stopped there as all css files are in it so i don't need to specify each css file
![screenshot 389 _li](https://user-images.githubusercontent.com/37622734/52517208-c0383300-2c5d-11e9-8a4e-a8efbcf3179f.jpg)

Reply:
ok. So when you give a relative path (something that does not start with `/`, for example `PyJudge/css`), the path is calculated from the current directory. In this case, since you are running `python server.py`, you are already inside the `PyJudge` folder. Hence, `css` would be enough for the root path.

relative paths can be quiet confusing for other people when they read your code. That's why I did the path join. In the file, there is already `dir_path`. This is the path to the folder `PyJudge`. You can try printing it's value and that should confirm it.

On top of that I joined a folder called `static`, which does not exist yet. That is so that I don't have to write another API / URL for javascript files when they come. I can just put them in the `static` folder under a sub-folder called `js`.

Reply:
Does that answer your question?

Reply:
So it means that `root=os.path.join(dir_path, 'static')` will create a folder called `static` inside `dir_path` which already present for our project and then when we need JS files we can create subfolder `js` inside folder `static`. Am I right?